packages/web/package.json

```diff
@@ -47,7 +47,8 @@
 		"react-redux": "^6.0.1",
 		"rheostat": "^2.1.1",
 		"url-search-params-polyfill": "^7.0.0",
-		"xdate": "^0.8.2"
+		"xdate": "^0.8.2",
+		"vue-types": "^1.3.3"
```

Comment:
@anik-ghosh-au7 What is the reason for this change?

Reply:
@bietkul because you mentioned it in the above comment - https://github.com/appbaseio/reactivesearch/pull/1696#discussion_r657367262